### PR TITLE
Add parameterized tests for edge negotiation tests

### PR DESCRIPTION
**Pull request by jcague**
**Description**

Add some parameterized tests for testing SDP negotiation.

- [x] It needs and includes Unit Tests

**Changes in Client or Server public APIs**

Not needed.

[] It includes documentation for these changes in `/doc`.

### CODE REVIEW

File: test/negotiation/index.js [hunk 1]
```diff
@@ -17,6 +17,99 @@ describeNegotiationTest('SDP negotiations started by Erizo', function(ctx) {
 });
 
 describeNegotiationTest('Conflicting SDP negotiation started by Erizo and Client', function(ctx) {
-  ctx.publishAndSubscribeStreamsStep();
-  ctx.publishAndSubscribeStreamsStep();
+  ctx.publishAndSubscribeStreamsStep([
+    'client-add-stream',
```
Comment (lodoyun) [inline test/negotiation/index.js:21]: I like the way this is arranged. It makes it easier to read 👍

**Review by lodoyun — APPROVED** [review]
This looks good to me 👍
